tests/text/test_bertscore.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any
+
+import numpy as np
+import pytest
+
+from torchmetrics.functional import bert_score
+from torchmetrics.text import BERTScore
+from torchmetrics.utilities.imports import _BERTSCORE_AVAILABLE
+
+preds = [
+    "28-year-old chef found dead in San Francisco mall",
+    "A 28-year-old chef who recently moved to San Francisco was "
+    "found dead in the staircase of a local shopping center.",
+    "The victim's brother said he cannot imagine anyone who would want to harm him,\"Finally, it went uphill again at "
+    'him."',
+]
+refs = [
+    "28-Year-Old Chef Found Dead at San Francisco Mall",
+    "A 28-year-old chef who had recently moved to San Francisco was found dead in the stairwell of a local mall this "
+    "week.",
+    "But the victim's brother says he can't think of anyone who would want to hurt him, saying, \"Things were finally "
+    'going well for him."',
+]
+
+
+def _assert_list(preds: Any, refs: Any, threshold: float = 1e-8):
+    """Assert two lists are equal."""
+    assert np.allclose(preds, refs, atol=threshold, equal_nan=True)
+
```

Comment:
for checking that accumulation in the modular version works
```suggestion
preds_batched = [preds[0:2], preds[2:]]
refs_batched = [target[0:2], target[2:]]
```